ams/KeysMigrator.cs

```diff
@@ -31,7 +31,16 @@ internal class KeysMigrator : BaseMigrator
         public override async Task MigrateAsync(CancellationToken cancellationToken)
         {
             _logger.LogInformation("Begin migration of keys for account: {name}", _keyOptions.AccountName);
-            var account = await GetMediaAccountAsync(_keyOptions.AccountName, cancellationToken);
+            MediaServicesAccountResource? account = null;
+            try
+            {
+                account = await GetMediaAccountAsync(_keyOptions.AccountName, cancellationToken);
+            }
+            catch (Exception)
+            {
+                _logger.LogError("No valid media account was found.");
+                return;
+            }
```

Comment:
This change is not needed either, it has nothing to do with the storage account support.